js/src/sdk/index.spec.ts

```diff
@@ -26,54 +26,70 @@
   });
 
   it("should handle 404 error gracefully", async () => {
-    const client = new Composio({ apiKey: COMPOSIO_API_KEY });
     const mock = new AxiosMockAdapter(axiosClient.instance);
-    mock.onGet("/api/v1/apps").reply(404, { detail: "Not found" });
+    const mockError = {
+      error: {
+        type: "NotFoundError",
+        name: "AppNotFoundError",
+        message: "Not found",
+      },
+    };
+    mock.onGet("/api/v1/apps").reply(404, mockError);
+
+    const client = new Composio({ apiKey: COMPOSIO_API_KEY });
 
     try {
-      await client.apps.list();
+      const apps = await client.apps.list();
+      console.log(apps);
```

Comment:
The `console.log(apps)` statement should be removed as it's not needed for the test and could clutter test output.